src/AcceptanceTestHelper/RuntimeSagaDefinitionReader.cs

```diff
@@ -80,4 +79,23 @@ public static SagaDefinition GetSagaDefinition(Type sagaType, BuildSqlDialect sq
             correlationProperty: correlationProperty,
             transitionalCorrelationProperty: transitional);
     }
+
+    static string GetSagaMetadataProperty(Type sagaType, Saga instance, string sqlSagaPropertyName, Func<SqlSagaAttribute, string> getSqlSagaAttributeValue)
+    {
+        if (sagaType.IsSubclassOfRawGeneric(typeof(SqlSaga<>)))
```

Comment:
so is the idea that we'll run the same set of tests twice, once for core saga and once for sql saga?

asking on this method, but that's general question for all tests in general